examples/pump-app/pump-common/gen/CHIPClusters.cpp

```diff
@@ -32,6 +32,8 @@
 
 #include <gen/CHIPClientCallbacks.h>
 
+using namespace chip;
```

Comment:
Why do you even need it? We open the namespace on line 79, can't you just move that upwards?

Reply:
acked